asana/asana.go

```diff
@@ -28,8 +28,18 @@ var defaultOptFields = map[string][]string{
 }
 
 type (
+	//Doer interface used for doing http calls.
+	//Use it as point of setting Auth header or custom status code error handling.
+	Doer interface {
+		Do(req *http.Request) (*http.Response, error)
+	}
+
+	//DoerFunc implements Doer interface.
+	//Allow to transform any appropriate function "f" to Doer instance: DoerFunc(f).
+	DoerFunc func(req *http.Request) (resp *http.Response, err error)
```

Comment:
Here, and elsewhere, the comments don't have a space after the two slashes. There should be a single space there.

See [here](https://gist.github.com/shurcooL/8241db3cfc2f21611ae8a780eeb2ede0#comments-for-humans-always-have-a-single-space-after-the-slashes) for rationale.